_pydevd_sys_monitoring/_pydevd_sys_monitoring.py

```diff
@@ -643,6 +661,7 @@ def _enable_return_tracing(code):
 # ENDIF
 # fmt: on
     # print('enable return tracing', code)
+    _ensure_monitoring()
```

Comment:
This function was being called prior to the debugger being setup as the sys.monitor